core/domain/exp_domain_test.py

```diff
@@ -1028,6 +1032,37 @@ def test_subtitled_html_validation(self):
             with self.swap(subtitled_html, 'audio_translations', 'not_list'):
                 subtitled_html.validate()
 
+    def test_get_trainable_states_dict(self):
```

Comment:
Would it be possible add a test where you add a new state that is trainable and see if it is present in the result dict?